app/src/main/java/com/swent/assos/ui/login/SignUpScreen.kt

```diff
@@ -9,42 +10,89 @@ import androidx.compose.runtime.getValue
 import androidx.compose.runtime.mutableStateOf
 import androidx.compose.runtime.remember
 import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
 import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.semantics.semantics
+import androidx.compose.ui.semantics.testTag
 import androidx.compose.ui.text.input.PasswordVisualTransformation
-import androidx.compose.ui.tooling.preview.Preview
 import androidx.hilt.navigation.compose.hiltViewModel
+import androidx.navigation.NavController
+import com.google.firebase.functions.FirebaseFunctions
+import com.google.firebase.functions.functions
+import com.swent.assos.model.navigation.Destinations
 import com.swent.assos.model.view.LoginViewModel
+import org.w3c.dom.Text
 
-@Preview
 @Composable
-fun SignUpScreen() {
+fun SignUpScreen(navController: NavController) {
   var email by remember { mutableStateOf("") }
   var password by remember { mutableStateOf("") }
   var confirmPassword by remember { mutableStateOf("") }
   val loginViewModel: LoginViewModel = hiltViewModel()
+  var error by remember { mutableStateOf("") }
 
   Column {
-    OutlinedTextField(value = email, onValueChange = { email = it }, label = { Text("Email") })
+    OutlinedTextField(
+        value = email,
+        onValueChange = { email = it },
+        label = { Text("Email") },
+        modifier = Modifier.semantics { testTag = "email" })
     OutlinedTextField(
         value = password,
         onValueChange = { password = it },
         label = { Text("Password") },
-        visualTransformation = PasswordVisualTransformation())
+        visualTransformation = PasswordVisualTransformation(),
+        modifier = Modifier.semantics { testTag = "password" })
```

Comment:
We use a different way to write testTags in the rest of the project, it is maybe a good idea to write it in the same way in the entire project. `Modifier.semantics { testTag = "password" }` would become `Modifier.testTag("password")`